### DISCUSSION TITLE
Content (or substitute) attribute is declared on multiple properties

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#12698] [Q&A] [answered]
When do you estimate to merge the PR https://github.com/kekekeks/XamlX/pull/92 in Avalonia?

**Comment by timunie** [comment]
It is merged? What am I missing?

**Reply by workgroupengineering** [reply]
In master e in 11.0.4 the xamlX is 1ff4bc251de45738d22cb5b173fbbc29fa923ec5
![immagine](https://github.com/AvaloniaUI/Avalonia/assets/12531229/4e7bc2d1-1f15-4d4c-a95c-f6086e93574c)

I have same issue as https://github.com/kekekeks/XamlX/issues/45

**Reply by workgroupengineering** [reply]
@timunie  Can I create PR to update XamlX submodule?

**Reply by maxkatz6** [reply, marked as answer]
@workgroupengineering yes, thank you
